Images/EPSSeparations/src/main/java/com/datalogics/pdfl/samples/EPSSeparations.java

```diff
@@ -157,17 +157,11 @@ else if(i.getColorantName() == "PANTONE 7467 C")
 
             for(SeparationPlate p : plates)
             {
-                boolean emptyFile = false;
-                if (p.getEPSOutput().getLength() == 0)
-                {
-                    emptyFile = true;
-                }
-
                 p.getEPSOutput().close();
 
-                if (emptyFile)
+                File f = new File("Complex-Pg" + (pgNum + 1) + "-" + p.getColorantName() + ".eps");
```

Comment:
This doesn't build because ``f`` is already used as the argument name for the method this is in: 
``public static void complexSeparationsExample(String f) throws Throwable``

Reply:
fixed